src/ontology/uberon-edit.obo

```diff
@@ -166585,18 +166587,27 @@ is_a: UBERON:0015228 ! circulatory organ
 [Term]
 id: UBERON:0015230
 name: dorsal vessel heart
-synonym: "heart" RELATED SENSU []
+def: "The caudal, pulsatile region of the dorsal vessel of the arthropod circulatory system." [FBbt:00003154, UBERON:cjm]
+synonym: "adult heart" NARROW SENSU [FBbt:00003154, NCBITaxon:50557]
+synonym: "embryonic/larval heart" NARROW SENSU [FBbt:00001675, NCBITaxon:50557]
+synonym: "heart" RELATED SENSU [NCBITaxon:50557]
 is_a: UBERON:0007100 ! primary circulatory organ
 relationship: part_of UBERON:0015231 {source="FBbt"} ! circulatory system dorsal vessel
+property_value: external_ontology_notes "FBbt has two terms that are both narrower than this, for adult heart, and embryonic/larval heart." xsd:string
 
 [Term]
 id: UBERON:0015231
 name: circulatory system dorsal vessel
-synonym: "adult dorsal vessel" RELATED [FBbt:00003152]
+def: "A tube that runs longitudinally through the thorax and abdomen, along the inside of the dorsal body wall and that forms the major structural component of the open circulatory system of arthropods." [UBERON:cjm]
+synonym: "adult dorsal vessel" NARROW SENSU [FBbt:00003152, NCBITaxon:50557]
+synonym: "adult dorsal vessel" NARROW [FBbt:00003152]
 synonym: "dorsal vessel" EXACT []
+synonym: "embryonic/larval dorsal vessel" NARROW SENSU [FBbt:00001673, NCBITaxon:50557]
```

Comment:
Now that FBbt has a stage-neutral term for that structure, I am not sure we need the narrow synonyms pointing to the adult and larval variants. I’d suggest replacing those by a single

```
synonym: "dorsal vessel" NARROW SENSU [FBbt:00058291, NCBITaxon:50557]
```